### ISSUE TITLE
Can't use locale emulator on a separate hard disk

### ISSUE THREAD

**Issue by thomastheswift** [xupefei/Locale-Emulator#465]
Launching games that don't work without locale emulator won't work on a separate plugged in hard disk for some reason. It works if I move the game over to my main hard disk or my other installed hard disk, but it won't work when it's a separate one plugged in via usb. Kinda sucks since I've got all my japanese games on a separate hard disk since they take such a large amount of space.

I was just wondering if this was an issue I'm just having, and if you know about this. Can something be done to make it work?

**Comment by xupefei** [comment]
I don't think it is caused by LE. Those games may have their own logic for checking installation folder and refuse to work after you have moved them.
